### PR TITLE
The return of the grade management

### PR DESCRIPTION
**Pull request by CrumbleZ**

### CODE REVIEW

File: app/config/services.yml [hunk 1]
```diff
@@ -11,10 +11,16 @@ services:
...
         class: AppBundle\Serializer\ReportNormalizer
-        arguments: ["@router", "@app.report_manager"]
+        arguments: ["@router", "@app.report_manager", "@app.user_manager"]
         tags:
             - { name: serializer.normalizer }
 
+    app.form.type.report:
+        class: AppBundle\Form\ReportType
+        arguments:
+            - '@app.user_manager'
```
Comment (maidmaid) [inline app/config/services.yml]: Doublequotes instead.

File: app/config/services.yml [hunk 2]
```diff
@@ -11,10 +11,16 @@ services:
...
+        tags:
+            - { name : form.type }
```
Comment (maidmaid) [inline app/config/services.yml]: Remove space between name and :

File: src/AppBundle/Form/ReportType.php [hunk 3]
```diff
@@ -42,7 +53,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('addressedTo', EntityType::class, [
                 'class' => 'AppBundle\Entity\User',
-                'choice_label' => 'username',
+                'choice_label' => function (User $user) {
+                    return $this->userManager->getFullName($user, true);
+                },
```
Comment (maidmaid) [inline src/AppBundle/Form/ReportType.php]: Anomynous function on online instead.

**Comment by CrumbleZ** [comment]
I suppose that the rank display in the left hand side of the UI should also be done in this PR ? Amirite ?

**Pull request merged**